src/hooks/v1/transactor/PayV1ProjectTx.ts

```diff
@@ -23,7 +23,21 @@ export function usePayV1ProjectTx(): TransactorInstance<{
       !contracts?.TicketBooth ||
       !terminal?.version
     ) {
-      txOpts?.onDone?.()
+      let missingParam = !transactor
+        ? 'transactor'
+        : !projectId
+        ? 'projectId'
+        : !contracts?.TicketBooth
+        ? 'contracts.TicketBooth'
+        : !terminal?.version
+        ? 'terminal.version'
+        : null
```

Comment:
I've seen a bunch of ternary in the code base and assumed this would be okay. I know there are a lot of opinions around nested ternary.

Reply:
Appreciate the comment! I like this approach.